src/masternode/meta.cpp

```diff
@@ -153,10 +153,44 @@ void CMasternodeMetaMan::RememberPlatformBan(const uint256& inv_hash, PlatformBa
     m_seen_platform_bans.insert(inv_hash, std::move(msg));
 }
 
+void CMasternodeMetaMan::AddUsedMasternode(const uint256& proTxHash)
+{
+    LOCK(cs);
+    // Only add if not already present (prevents duplicates)
+    if (m_used_masternodes_set.insert(proTxHash).second) {
+        m_used_masternodes.push_back(proTxHash);
+    }
+}
+
+void CMasternodeMetaMan::RemoveUsedMasternodes(size_t nCount)
```

Comment:
nit: nCount -> count


```suggestion
void CMasternodeMetaMan::RemoveUsedMasternodes(size_t count)
```